### PR TITLE
Open With Default Program

### PR DESCRIPTION
**Pull request by fire**
*Bugsquad edit: This closes https://github.com/godotengine/godot-proposals/issues/912. This partially addresses https://github.com/godotengine/godot-proposals/issues/328.*

CICD build.

### CODE REVIEW

**Comment by fire** [comment]
image

**Comment by Calinou** [comment]
"Open Original" sounds a bit strange. How about "Open With Default Program"?

**Comment by fire** [comment]
Sure, that's an easy change but it's a bit wordy. Will do a change later today.

**Comment by akien-mga** [comment]
I understand that you're using the `3.2` branch in production, but such non-critical changes should definitely be merged in `master` first, and eventually backported to `3.2` if relevant.

**Comment by fire** [comment]
Created master pr.

**Comment by fire** [comment]
Will reduce the number of 3.2 prs.
